woocommerce/woocommerce-android
# [Better Tablet Products] Remove margin from the toolbar from product details fragment. Toolbar to Variation details fragment

Closes: #10979
Closes: #10977


### Description

The PR:
* Removes margins from the toolbar of the product details screen
* Adds toolbar to VariationDetailFragment.kt
* Fixes back navigation (at least makes it possible) for the cases when 2 panes mode switched to 1 pane

### Testing instructions

* Try the changes mentioned above (check the video for better understanding)
* Make sure no regressions introduced when the FF is OFF


### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/6df0e706-65c2-46cb-b9f4-5baa18748f12
